src/contracts/security/security.service.ts

```diff
@@ -161,6 +171,7 @@ export class SecurityService {
   @OneAtTime()
   public async pauseDeposits(
```

Comment:
In this service we also need to modify methods:
- signDepositData
- signPauseData